docs/modules/ROOT/pages/index.adoc

```diff
@@ -1 +1,49 @@
-= INDEX
\ No newline at end of file
+:source-highlighter: highlight.js
+:highlightjs-languages: bash
+
+= Substrate Parachain Runtimes
+
+A collection of runtimes that describe parachains with different purposes.
+
+IMPORTANT: This repo contains code in its early stages. Please be aware that it may change rapidly and was not yet audited.
```

Comment:
is not yet audited?

Reply:
Our pallets are not audited by OpenZeppelin.
That's what Jonathan mentioned on our call on Tuesday.

Reply:
no no, i meant `is not yet audited` rather than `was not yet audited` as a grammar fix. But I'm not native :)

Reply:
Fixed :)